.github/workflows/build-release-packages.yml

```diff
@@ -43,6 +54,16 @@ jobs:
           # only ever be a tag
           INPUT_VERSION: ${{ inputs.version || github.ref_name }}
 
+      - name: Set bucket name
+        id: check-env
+        run: |
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            echo "IS_TEST=true" >> $GITHUB_OUTPUT
+          else
+            echo "IS_TEST=false" >> $GITHUB_OUTPUT
+          fi
+        shell: bash
+
```

Comment:
Check if whether it is a pull request or it's an actual release (manual or automated) and output true or false. This step purpose is to know if we should use a `test` bucket or a `prod` bucket